content/attachments/c0ceaa82-e37c-4b86-a766-59751c6759db/c0ceaa82-e37c-4b86-a766-59751c6759db-příloha.ttl

```diff
@@ -29,7 +28,7 @@
   <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/scheme> <https://slovník.gov.cz/legislativní/sbírka/99/2004/glosář>;
   <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/vocabulary> <https://slovník.gov.cz/legislativní/sbírka/99/2004> .
 
-<http://onto.fel.cvut.cz/ontologies/application/ontoGrapher-bcc51c2f-0e49-41d1-96bb-531fc093f98d>
+<http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/link/instance-bcc51c2f-0e49-41d1-96bb-531fc093f98d>
```

Comment:
I suggest using a more specific IRI format for links

Reply:
no opinion on this

Reply:
@karelklima not sure what exactly you mean here. Could you provide some example?